plugins/modules/win_domain_computer.ps1

```diff
@@ -146,6 +157,73 @@ Function Add-ConstructedState($desired_state) {
   $result.changed = $true
 }
 
+Function Invoke-OfflineDomainJoin {
+  [CmdletBinding(SupportsShouldProcess=$true)]
+  param(
+    [Parameter(Mandatory=$true)]
+    [System.Collections.IDictionary]
+    $desired_state ,
+
+    [Parameter()]
```

Comment:
Minor nit, `[Parameter()]` isn't really needed as that's the default for an advanced function.

Reply:
I'm aware; I prefer it personally for consistency, but I don't feel strongly enough about it to keep it if you prefer it omitted.

Reply:
No worries, just thought I would mention it.